src/App.js

```diff
@@ -1,25 +1,10 @@
 import React from 'react';
-import logo from './logo.svg';
+import FamilyTree from './FamilyTree.jsx'
```

Comment:
Not sure about `.jsx` files, but I know that you don't need to extension for `.js` files.

Reply:
If there's a folder with the component name FamilyTree you can just do 'import FamilyTree from './FamilyTree' 

We should probably also look into reorganizing folder structure w/one per component 
![image](https://user-images.githubusercontent.com/8893880/79544945-ab66e480-8044-11ea-8bba-ce47256f2343.png)

Here's an example of how we did folder structure for portal https://github.com/acmucsd/membership-portal-ui/tree/master/src

Reply:
Sounds good, I have re-structured the folders. For the import statement, seems like it can't resolve directories on my end, so I ended up doing `import FamilyTree from "../FamilyTree/FamilyTree"`, etc.